workers/tests/integrationtests/test_oqt.py

```diff
@@ -14,157 +20,114 @@
-                "building_count",
-                dataset=self.dataset,
-                feature_id=self.feature_id,
-            )
+        param = IndicatorDatabase(
+            name=self.name,
+            layerName=self.layer_name,
+            dataset=self.dataset,
+            featureId=self.feature_id,
         )
-        self.assertIsNotNone(indicator.result.label)
-        self.assertIsNotNone(indicator.result.value)
-        self.assertIsNotNone(indicator.result.description)
-        self.assertIsNotNone(indicator.result.svg)
+        indicator = asyncio.run(oqt.create_indicator(param))
+        self.run_tests(indicator)
 
     @oqt_vcr.use_cassette()
     def test_create_indicator_dataset_custom_fid_field_int(self):
-        indicator = asyncio.run(
-            oqt.create_indicator(
-                "GhsPopComparisonBuildings",
-                "building_count",
-                dataset=self.dataset,
-                feature_id=self.feature_id,
-                fid_field=self.fid_field,
-            )
+        param = IndicatorDatabase(
+            name=self.name,
+            layerName=self.layer_name,
+            dataset=self.dataset,
+            featureId=self.feature_id,
+            fidField=self.fid_field,
         )
-        self.assertIsNotNone(indicator.result.label)
-        self.assertIsNotNone(indicator.result.value)
-        self.assertIsNotNone(indicator.result.description)
-        self.assertIsNotNone(indicator.result.svg)
+        indicator = asyncio.run(oqt.create_indicator(param))
+        self.run_tests(indicator)
 
     @oqt_vcr.use_cassette()
     def test_create_indicator_dataset_custom_fid_field_str(self):
-        indicator = asyncio.run(
-            oqt.create_indicator(
-                "GhsPopComparisonBuildings",
-                "building_count",
-                dataset=self.dataset,
-                feature_id="Heidelberg",
-                fid_field="name",
-            )
+        param = IndicatorDatabase(
+            name=self.name,
+            layerName=self.layer_name,
+            dataset=self.dataset,
+            featureId="Heidelberg",
+            fidField="name",
         )
-        self.assertIsNotNone(indicator.result.label)
-        self.assertIsNotNone(indicator.result.value)
-        self.assertIsNotNone(indicator.result.description)
-        self.assertIsNotNone(indicator.result.svg)
+        indicator = asyncio.run(oqt.create_indicator(param))
+        self.run_tests(indicator)
 
     @oqt_vcr.use_cassette()
-    def test_create_indicator_from_database_invalid_arguments(self):
-        indicator = asyncio.run(
-            oqt.create_indicator(
-                "GhsPopComparisonBuildings",
-                "building_count",
-                dataset=self.dataset,
-                feature_id=self.feature_id,
-                feature=self.feature,
-            )
+    def test_create_report_bpolys(self):
+        """Test creating indicator from scratch."""
```

Comment:
```suggestion
        """Test creating indicator from scratch using the 'bpolys' parameter."""
```
Something like that to make it clear that it doesn't use any force or failure method.